src/scala/com/github/johnynek/bazel_deps/Label.scala

```diff
@@ -59,6 +61,7 @@ object Label {
 
     val artName = lang match {
       case Language.Java => m.toTargetName
+      case Language.Kotlin => m.toTargetName
```

Comment:
can merge with the above: `case Language.Java | Language.Kotlin =>`

Reply:
👍 had a similar thought. Couldn't remember the syntax. Been writing more kotlin than scala lately. Tried `||` then `or` then forgot to google..